src/rtf/include/rtf/Asserter.h

```diff
@@ -37,31 +37,31 @@ class RTF::Asserter {
      * given message.
      * @param msg The corresponding failure message
      */
-    static RTF_API void fail(RTF::TestMessage msg);
+    RTF_API static void fail RTF_NORETURN (RTF::TestMessage msg);
```

Comment:
While allowed by the standard, the placement of this `RTF_NORETURN` is unconventional. I think t remember that there was a reason for this (`RTF_API` is a GCC-style attribute and `RTF_NORETURN` is a C++11-attribute, and this combination is affected by some bug in GCC5/6 ) but it would be good to remind what reason there is for this placement in the PR description.